rust/src/utils.rs

```diff
@@ -231,6 +231,49 @@ pub trait DeserializeEmbeddedGroup {
     ) -> Result<Self, DeserializeError> where Self: Sized;
 }
 
+pub struct CBORReadLen {
```

Comment:
cddl-codegen was updated to check array/map lengths during deserializaiton so this stuff was added.